docs/lib/analytics/fragments/android/getting-started/40_record.md

```diff
@@ -0,0 +1,21 @@
+To record an event, specify your event using `BasicAnalyticsEvent` and call `Amplify.Analytics.recordEvent()`
+
+```java
+BasicAnalyticsEvent event = new BasicAnalyticsEvent(
```

Comment:
API is wrong - Class is `AnalyticsEvent`, I think.

Reply:
Changed